src/doppler-types.ts

```diff
@@ -0,0 +1,10 @@
+// TODO: move it a common place if it will be reused
```

Comment:
Esta línea creo que ya no es necesaria.

Reply:
> This is weird, this later I check it
> 
> ![shopy](https://user-images.githubusercontent.com/46735526/60031916-f1008e00-967b-11e9-8c81-0cf54f248526.gif)



> This is weird, this later I check it
> 
> ![shopy](https://user-images.githubusercontent.com/46735526/60031916-f1008e00-967b-11e9-8c81-0cf54f248526.gif)

I don't see how to reproduce this, can you tell me please?